### DISCUSSION TITLE
World link Expired !

### DISCUSSION THREAD

**Discussion by AyanMCAniamtor** [jacobfortune/Songs-Of-War-Addon#8] [Q&A] [answered]
The link you provide to download mcworld is expired on mediafire , I suggest using google drive link instead of media fire or else please provide a new link to download

**Comment by lesleyespire** [comment]
Yeah it probably is, just been a long while since anyone checked out this old repository

**Reply by lesleyespire** [reply, marked as answer]
I have two links;
Season 2 World Songs Of War Server Launch World; https://www.mediafire.com/file/r3vho55qnt02c04/S2_LAUNCH.mcworld/file 
&
Ardonia(2). https://www.mediafire.com/file/0u985g9kxnagpeu/Ardonia%25282%2529.mcworld/file
